main.css

```diff
@@ -0,0 +1,173 @@
+body {
+    font-family: 'Wix Madefor Text', sans-serif;
+    height: 100vh;
+    overflow: hidden;
+}
+
+li {
+    list-style-type: none;
+}
+
+/* header */
+.header-wrapper {
+    width: 100%;
+    height: 60px;
+    padding-left: 35px;
+    padding-right: 35px;
+    border-bottom: 1px solid var(--border);
+
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
+.header__nav-wrapper {
+    display: flex;
+    gap: 30px;
+}
+
+.calendar-logo {
+    width: 40px;
+    height: 40px;
+}
+
+.btn {
+    justify-content: center;
+    align-items: center;
+    width: auto;
+    height: 40px;
+    border: unset;
+    background-color: #FFFFFF;
+    cursor: pointer;
+    color: var(--primary);
+    font-size: 16px;
+    font-weight: 600;
+    text-decoration: none;
+}
+
+.btn--border {
+    border: 1px solid var(--border);
+    border-radius: .25rem;
+}
+
+.btn--round:hover {
+    height: 50px;
+    width: 50px;
+    border-radius: 50%;
+}
+
+.btn:hover,
+.btn:focus {
+    background-color: var(--hover);
+}
+
+/* main wrapper */
+.main-wrapper{
+    display: flex;
+    gap: 5%;
+    margin-right: 30px;
+}
+
+/* side wrapper */
+.side-wrapper {
+    width: max-content;
+}
+
+.side-calendar__header {
+    display: flex;
+    justify-content: space-between;
+}
+
+.side-calendar__body{
+    font-size: 12px;
+}
+
+.side-calendar__week,
+.side-calendar__list {
+    display: grid;
```

Comment:
I don't think grid is needed here. You should make this side calendar the same like large calendar - have one parent ul with `display: grid` and all calendar cells as it's child.